django_socio_grpc/generics.py

```diff
@@ -158,7 +160,22 @@ def get_serializer_context(self):
     def filter_queryset(self, queryset):
```

Comment:
filter_queryset should only return a queryset not resolve any calls, so this should not have any use